kustomization/components/samba-upload/deployment.yml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: samba
+spec:
+  revisionHistoryLimit: 4
```

Comment:
aside: we should maybe add this to all our stuff; I don't think we're consistent.

Reply:
Another feature request for common patches...maybe we can reference patches by urls, like with components?